src/main/java/com/traveltime/sdk/dto/requests/proto/Transportation.java

```diff
@@ -11,20 +11,14 @@ public interface Transportation {
     Transportation WALKING_FERRY = Modes.WALKING_FERRY;
     Transportation CYCLING_FERRY = Modes.CYCLING_FERRY;
     Transportation DRIVING_FERRY = Modes.DRIVING_FERRY;
-    Transportation WALKING = Modes.WALKING;
-    Transportation CYCLING = Modes.CYCLING;
-    Transportation DRIVING = Modes.DRIVING;
 
     @Getter
     @AllArgsConstructor
     enum Modes implements Transportation {
         PUBLIC_TRANSPORT("pt", 0),
         WALKING_FERRY("walking+ferry", 7),
         CYCLING_FERRY("cycling+ferry", 6),
-        DRIVING_FERRY("driving+ferry", 3),
-        WALKING("walking", 4),
-        CYCLING("cycling", 5),
-        DRIVING("driving", 1);
```

Comment:
Do not remove transportation modes without ferries, they still does exist.

Reply:
rolled back these changes